src/test/java/amazin/BaseTest.java

```diff
@@ -29,61 +30,57 @@ public static void tearDownAfterClass() throws Exception {
     @Before
     public void setUp() throws Exception {
         book = new Book(DEFAULT_ID, DEFAULT_NAME, DEFAULT_DESCRIPTION, DEFAULT_ISBN, DEFAULT_PICTURE, DEFAULT_AUTHOR,
-                DEFAULT_PUBLISHER, DEFAULT_SKU, DEFAULT_INVENTORY, DEFAULT_PRICE);
+                DEFAULT_PUBLISHER, DEFAULT_INVENTORY, DEFAULT_PRICE);
     }
 
     @After
     public void tearDown() throws Exception {
     }
 
     protected void testAllFieldsDefaultExcept(Fields.Book field) {
-        for(Fields.Book f : Fields.Book.values()) {
+        for (Fields.Book f : Fields.Book.values()) {
```

Comment:
what is this useful for

Reply:
This is done automatically by a linting plugin installed in vs code to maintain consistency.